web/src/main/java/com/futurewei/alcor/web/entity/port/PortEntity.java

```diff
@@ -108,6 +109,7 @@
     @JsonProperty("revision_number")
     private int revisionNumber;
 
+    @JsonIgnore
```

Comment:
This is a change from other PR. It is not supposed to be here.

Reply:
fixed in  #310

Reply:
Correct. As this file has been merged to master, it should be removed from this PR.

Reply:
fixed in 841a344